features/promotion/managing_promotions/adding_promotion.feature

```diff
@@ -0,0 +1,18 @@
+@managing_promotions
+Feature: Adding a new promotion
+    In order to give possibility to pay less for some goods
+    As an Administrator
+    I want to add a new promotion to the registry
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Adding a new promotion
```

Comment:
Sure :+1: But for now, as we don't have actions adding implemented yet, it still checks some basic things (specification of name and code). Maybe when we add actions feature to new backend we could modify this scenario to check promotion with one action?